cpp/autosar/src/rules/A4-7-1/IntegerExpressionLeadToDataLoss.ql

```diff
@@ -30,5 +30,7 @@ where
   not e instanceof MulExpr and
   // Not covered by this query - overflow/underflow in division is rare
```

Comment:
quick question, do we need to consider operand types for this query? in the fp report the wording is "regardless of the arguments involved" and its mentioned that the expectation is for integral types to not be an issue, but floating point types still could be?

Reply:
This query only targets integers. I will have look at where this is constrainted.

Reply:
Already address in PR #490